data/mods/My_Sweet_Cataclysm/sweet_monster_factions.json

```diff
@@ -57,5 +57,18 @@
     "name": "licorice_snakes",
     "neutral": [ "bot", "plant", "nether", "human", "crackers", "cookies" ],
     "by_mood": [ "insect", "player", "animal", "gummy_bears", "chewgum" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "pumpkin_man",
+    "hate": [ "bot", "nether", "human", "insect" ],
+    "by_mood": [ "player", "animal", "crackers", "cookies", "gummy_bears", "chewgum" ],
+    "friendly": [ "plant" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "candy_cane",
+    "hate": [ "animal", "crackers", "cookies", "gummy_bears", "chewgum", "bot", "nether", "human", "insect", "plant" ],
```

Comment:
Why does it hate bots, and nether and humans? You should reuse the same kind of relations as the other factions:
Candies are only hostile to similar candies because they feed on them, zombies, robots, nether, humans etc don't matter because they are not food.